numpy/core/src/multiarray/textreading/parser_config.h

```diff
@@ -2,6 +2,10 @@
 #ifndef NUMPY_CORE_SRC_MULTIARRAY_TEXTREADING_PARSER_CONFIG_H_
 #define NUMPY_CORE_SRC_MULTIARRAY_TEXTREADING_PARSER_CONFIG_H_
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
 #include <stdbool.h>
```

Comment:
the include below should not be guarded by `extern "C"`. Only guard declarations.